code/modules/cargo/packs/research.dm

```diff
@@ -2,76 +2,62 @@
 	group = "Science"
 	crate_type = /obj/structure/closet/crate/science
 
+/datum/supply_pack/science/faction_tech
+	faction_discount = 0
+	faction_locked = TRUE
+	cost = 1000
+	small_item = TRUE
+
 /datum/supply_pack/science/faction_tech/nanotrasen
 	name = "NanoTrasen Equipment Design Drive"
 	desc = "Contains one NanoTrasen Equipment Design Drive."
-	cost = 1000
-	small_item = TRUE
 	faction = /datum/faction/nt
 	contains = list(/obj/item/faction_tech/nanotrasen)
 
 /datum/supply_pack/science/faction_tech/hardliners
 	name = "Hardliners Equipment Design Drive"
 	desc = "Contains one Hardliners Equipment Design Drive."
-	cost = 1000
-	small_item = TRUE
 	faction = /datum/faction/syndicate/hardliners
-	contains = list(/obj/item/faction_tech/hardliners, /obj/item/faction_tech/scarborough)
+	contains = list(/obj/item/faction_tech/hardliners, /obj/item/faction_tech/gorlex_marauders, /obj/item/faction_tech/scarborough)
 
 /datum/supply_pack/science/faction_tech/gorlex_republic
 	name = "New Gorlex Republic Equipment Design Drive"
 	desc = "Contains one New Gorlex Republic Equipment Design Drive."
-	cost = 1000
```

Comment:
supply packs that include more than one design kit should cost extra or be split up into different packs